xarray/core/dataarray.py

```diff
@@ -3819,6 +3819,209 @@ def idxmax(
             keep_attrs=keep_attrs,
         )
 
+    def argmin(
+        self,
+        dim: Union[Hashable, Sequence[Hashable]] = None,
+        axis: Union[int, None] = None,
```

Comment:
nit: IIUC this could be
```suggestion
        axis: int = None,
```
like `keep_attrs` and `skipna` below